universum/modules/api_support.py

```diff
@@ -45,5 +46,10 @@ def get_environment_settings(self) -> Dict[str, str]:
     def add_file_diff(self, entry: str) -> None:
         self._set_entry("DIFF", entry)
 
+    def register_file_diff_failure(self) -> None:
```

Comment:
Considering there is only one symmetric function (get_file_diff), it is better to keep setting the value and error status within single function.